bloom-instance/s3.tf

```diff
@@ -10,6 +10,15 @@ resource "aws_s3_bucket" "static_content" {
   bucket_prefix = "${var.name_prefix}-static-content"
 }
 
+resource "aws_s3_bucket_public_access_block" "public_uploads" {
+  bucket = aws_s3_bucket.public_uploads.id
+
+  block_public_acls       = false
```

Comment:
The title of the PR  is 'block' but the attrs are all `false` on blocking public acls- do these need to be `true`?

Reply:
I think I understand- this is explicit unblocking.